README.md

```diff
@@ -141,6 +143,7 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
 | <a href="https://bamdesign.gumroad.com/l/bamillu"><img width='700' src='https://user-images.githubusercontent.com/74038190/184643153-3bc26335-d868-494f-8818-8201949fc44b.png' alt="Bam_3D_free_illustrations" /></a> | [Bam 3D](https://bamdesign.gumroad.com/l/bamillu)  <br><br> ✔ Free <br><br>Format <br> • PNG<br><br> ✌️Author <br> BAM Design                                                                                            |
 | <a href="https://bigheads.io/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808185-47ad4564-eb78-4b13-87f9-a8f973940cd1.png' alt="bigHeads_free_illustrations" /></a> | [Big Heads](https://bigheads.io/)  <br><br> ✔ Free <br><br>Format <br>• SVG<br><br> ✌️Author <br> Big Heads |
 | <a href="https://www.getillustrations.com/illustration-pack/blues-dual-tone-free-illustrations"><img width='700' src='https://user-images.githubusercontent.com/74038190/184643324-bb155fb8-1b59-41e4-828d-3512f02b667a.png' alt="Blues_free_illustrations" /></a> | [Blues](https://www.getillustrations.com/illustration-pack/blues-dual-tone-free-illustrations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• AI<br>• SVG<br>• AI<br><br> ✌️Author <br> Getillustrations                                                        |
+| <a href="https://boringavatars.com"><img width='700' src='https://raw.githubusercontent.com/Sn4pe/Awesome-Illustrations-4Projects/main/assets/boring-avatar.png' alt="boring_avatar" /></a> | [Boring Avatars](https://boringavatars.com)  <br><br> ✔ Free <br><br>Format <br> • SVG<br><br> ✌️Author <br> hihayk and josepmartins                                                        |
```

Comment:
- Use this screenshot URL `https://user-images.githubusercontent.com/74038190/194601571-dfae3f4a-05c2-478d-b445-7eace11eac59.png` &nbsp; since the visibility of the asset is very bad in the current screenshot

- Use `Hayk An & Josep Martins` as the author's value

- Use alt text as `Boring_Avatars_free_illustrations`